.rubocop.yml

```diff
@@ -12,18 +12,16 @@ AllCops:
   SuggestExtensions: false
   Exclude:
   - "vendor/**/*"
-  - "spec/support/repo/bin/tapioca"
 
-Style/CaseEquality:
+Gemspec/RequireMFA:
```

Comment:
This should not be here. This cop is enable disabled in our style guide. You should disable the `NewCops`.